README.md

```diff
@@ -88,7 +88,15 @@ The `testDF` function can be used to test the equality of dataframes. It uses `d
 
 #### `testGGPlot`
 
-The `testGGPlot` function can be used to test the equality of `GGPlot`s. Aside from the usual `description`, `generated` and `expected` arguments it has 5 optional arguments: `test_data`, `test_aes`, `test_geom`, `test_label` and `test_scale`. These are used to determine whether that particular layer should be tested or not. `test_data`, `test_aes` and `test_geom` are true by default, `test_label` and `test_scale` are false by default.
+The `testGGPlot` function can be used to test the equality of `GGPlot`s. Aside from the usual `description`, `generated` and `expected` arguments it has some optional arguments: 
+- `show_expected = TRUE` A logical value indicating whether the solution plot should be showed to the student if the testGGPlot function failes.
```

Comment:
I'm not sure if this is something we want to support. I get not wanting to give the solution to the student, but the whole point of Dodona is being able to compare your own output with the expected output. @bmesuere opinion?

Reply:
The Python judge also gas such aan option. Not used very often, but there are use cases.

Reply:
If it is enabled by default, I'm not against such switch.

Reply:
OK, so consensus seems to be that keeping the switch is fine. Maybe add a small line to the docs discouraging its use?

Reply:
Just for clarification:
The reason why we decided to have such an option is that in many of the exercises you have to draw a plot from which you have to conclude something. And because there is not 1 right solution we would not like to kill the creativity of the students by showing them the solution. Of course the students need feedback and that's why we are now trying to improve the textual feedback an flexibility of the ggplot testfunction. We don't know if this approach will work but I think its worth trying. 
Any feedback or conclusions from previous experiences with this kind of feedback are welcome!